src/main/target/common_defaults_post.h

```diff
@@ -659,5 +659,14 @@
 #ifndef MAX_SUPPORTED_MOTORS
 #define MAX_SUPPORTED_MOTORS 8
 #endif
+#ifndef MAX_SUPPORTED_MOTOR_PORTS
+#define MAX_SUPPORTED_MOTOR_PORTS 4 // Max direct dshot port groups, limited by number of usable timer (TIM1 and TIM8) x number of channels per timer (4), 3 is enough to cover motor pins on GPIOA, GPIOB and GPIOC.
```

Comment:
Does it make sense to have this per-target configurable, or should this just be a fixed define? (Per - target configurable settings will become obsolete with Unified Targets anyway.)

Reply:
This depends on the maximum number of ports that might be used on a given architecture. So with unified targets it would be specified by architecture and still makes sense. 4 should cover all cases for now.

Reply:
I'd move that out of here and into the bitbanged Dshot units - no point having one more thing that we have to support being not what we expect it to be.